src/dbus/instropect.xml

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+  <interface name="io.openthread.BordeRrouter">
+    <method name="Scan">
+      <arg name="scan_result" type="a(taytayqqyyyyy)" direction="out"/> 
```

Comment:
where do we want to host comment to explain the meaning of results returned?

Reply:
How about follow comment style as in spinel.h?
e.g. https://github.com/openthread/openthread/blob/master/src/ncp/spinel.h#L1645